example/simple-server/simple-server.go

```diff
@@ -55,6 +55,7 @@ func plain(listenAddr string) {
 func withZiti(service string) {
 	listener, err := ziti.NewContext().Listen(service)
 	if err != nil {
+		fmt.Printf("Error binding service %+v\n", err)
```

Comment:
is the \n really needed? shouldn't we strive for one line?

Reply:
You need `\n` with `Printf`. `Printf` doesn't add a newline, `Println` does.

Reply:
It's because it's not a log stmt. Without the \n there's no new line. It's an example program, so I think either fmt or log is ok here.